README.md

```diff
@@ -171,7 +171,7 @@ Limit one VDO `storage_volume` per `storage_pool`.
 Underlying volume has to be at least 9 GB (bare minimum is around 5 GiB).
 
 ##### `compression`
-This specifies whether or not the Virtual Data Optimizer (VDO) will be used.
+This specifies whether the Virtual Data Optimizer (VDO) will be used.
```

Comment:
I would prefer "This specifies that the pool . . ." - I prefer indicative rather than conditional or subjunctive

Reply:
I guess the same https://github.com/linux-system-roles/storage/pull/230#discussion_r716107461 applies here - so whether is more appropriate - but I'm not sure what happens if you specify `compression: false` on an existing volume/pool that is using VDO compression . . .